templates/_macros/utils.twig

```diff
@@ -66,3 +66,135 @@
             height="{{ image.height }}">
     {% endif %}
 {% endmacro %}
+
+{#
+ |------------------------------------------------------------------------------
+ | Pagination for search
+ |------------------------------------------------------------------------------
+#}
+{% macro pagination(pageInfo, nearbyPagesLimit = 4) %}
+    {% set total = pageInfo.totalPages %}
+    {% set current = pageInfo.currentPage %}
+    {% spaceless %}
```

Comment:
Is there a particular reason why you use `spaceless` here?

Reply:
No specific reason, I think I was testing something and forgot to remove it. I have removed the spaceless.